doc/changelog.rst

```diff
@@ -9,11 +9,12 @@ Version 0.5
 **Release date**: yyyy-mm-dd
 
 **Changes**:
-
+* Add copy method to fatiando.mesher.
```

Comment:
Could you please change this to the following?

    Add a copy method to ``fatiando.mesher`` objects.